README.md

```diff
@@ -157,9 +157,12 @@ point), skipping through all build failures. Once a build finally succeeds, it
 will prompt me to manually check for a segfault and use my input to decide
 whether the current revision is good or bad.
 
-Keep in mind however that this is very early stages. Barely anything is
-documented. I built this to scratch my own itch, and I continue developing it
-whenever I need some feature.
+## Installation
```

Comment:
"Installation" probably isn't right here, since its not about permanent installation. Maybe "How to try it" or something similar.

Reply:
> "Installation" probably isn't right here, since its not about permanent installation.

Yeah, I noticed that, too when writing that title.

> Maybe "How to try it" or something similar.

I think "How to try" too much overlaps with the "how to use" instructions above. I'll go for "How to get nix-bisect" for now, although I'm not entirely happy with that, either. If anyone can think of a better section title, please let me know.

Reply:
I kind of prefer "how to try" to "how to get nix-bisect", but either works.